internal/db/schema/migrations/oss/postgres/48/03_create_kms_key_destructions.up.sql

```diff
@@ -0,0 +1,40 @@
+begin;
+
+  create table kms_key_destruction_status_enm (
+    name text primary key
+      constraint only_predefined_values_allowed
+      check(name in ('pending', 'running', 'completed', 'failed'))
+  );
+  comment on table kms_key_destruction_status_enm is
+    'Table holding valid statuses a key destruction can take';
+
+  insert into kms_key_destruction_status_enm (name)
+  values
+    ('pending'),
+    ('running'),
+    ('completed'),
+    ('failed');
+
+  create trigger 
+    immutable_columns
+  before
+  update on kms_key_destruction_status_enm
+    for each row execute procedure immutable_columns('name');
+
+  create table kms_key_destruction (
+    private_id wt_private_id primary key,
+    key_id wt_private_id not null, -- Note: not a foreign key, outlives original reference
+    activation_time wt_timestamp not null,
```

Comment:
What does `activation_time` mean? Is it the time the key was activated or the time the destruction of the key was activated? Is the value related to `state`?

Reply:
It is the time the key was originally activated, i.e. when the key was created. This is meant to be used by a person who wants to track exactly from and to when a key was in the various states, `active`, `deactivated` and `destroyed`.